tests/restore.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "Usage: $0 [dbg^|rel] [x86^|amd64^|arm] [-?]"
+    exit -8
+}
+
+
+source $(cd "$(dirname "$0")"; pwd -P)/testenv.sh
+
+if [ -z ${__CoreRT_BuildOS} ]; then
```

Comment:
@kangaroo, the scenario I am describing below about coreclr tests, involves exporting global variables when run with source .sh. So the variables with CoreRT_ prefix are not internal in the coreclr test run scenario.